File: src/main/java/bot/util/settings/EN.java
package bot.util.settings;

import bot.enums.Language;
import bot.run.AbitVTBot;
import bot.util.CommandsUtil;
import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
import org.telegram.telegrambots.meta.api.objects.Message;
import org.telegram.telegrambots.meta.api.objects.Update;
import org.telegram.telegrambots.meta.exceptions.TelegramApiException;

public class EN implements Setting{
    private final AbitVTBot bot;

    public EN(AbitVTBot bot) {
        this.bot = bot;
    }

    @Override
    public void execute(Update update) throws TelegramApiException {
        String chatId = CommandsUtil.getChatId(update);

        SendMessage sendMessage = new SendMessage();
        sendMessage.setChatId(chatId);
        sendMessage.setText("Language was changed to english");

        bot.getSql().updateLanguage(chatId, Language.EN);

        bot.execute(sendMessage);
    }
}

File: src/main/java/bot/util/settings/EN.kt
package bot.util.settings

import bot.enums.Language
import bot.run.AbitVTBot
import bot.util.CommandsUtil.getChatId
import org.telegram.telegrambots.meta.api.methods.send.SendMessage
import org.telegram.telegrambots.meta.api.objects.Update


class EN(private val bot: AbitVTBot) : Setting {

    override fun execute(update: Update?) {
        val chatId = getChatId(update!!)
        val sendMessage = SendMessage()
        sendMessage.chatId = chatId
        sendMessage.text = "Language was changed to english"
        bot.sqlFunctions.updateLanguage(chatId, Language.EN)
        bot.execute(sendMessage)
    }
}